### PR TITLE
Added birthday greeting card

### PR DESCRIPTION
**Pull request by kumarankm**
## Related Issue
  - Created a Birthday Greeting card
  
Closes: #750 

### Describe the changes you've made
I have created a greeting card with a birthday song, ballon effects, and card opening effect.

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.
Untitled

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -910,7 +914,8 @@ let projectData = [
     projectName: "Typing Practice",
     projectImage: "assets/GIFs/TypingSpeed.gif",
     projectUrl: "public/TypingPractice.html",
-  },
+  };
```
Comment (991rajat) [inline script.js:917]: Remove the semicolon Please

**Review by 991rajat — APPROVED** [review]
Reduce the size of gif also please

**Comment by kumarankm** [comment]
> Reduce the size of gif also please

@991rajat  Made necessary changes :)

**Review by 991rajat — APPROVED** [review]
Looks Good.

**Comment by 991rajat** [comment]
Thanks for this awesome contribution. 🤩 Keep Contributing.

**Pull request merged**